js/api/ParseServer.js

```diff
@@ -0,0 +1,38 @@
+import { Platform } from 'react-native'
+import Parse from 'parse/react-native'
+import Store from '../data/Store'
+
+export function connectToParseServer(server) {
+  // Sync the Store variable and the new passed value, if any
+  if (!server) {
+    server = Store.server
+  } else {
+    Store.server = server
+  }
+
+  // Connect to the specified Parse server
+  switch (Store.server) {
+    case 'local': connectToLocalServer(); break;
+    case 'remote-test': connectToRemoteTestServer(); break;
+    default: connectToCustomServer(server); break;
+  }
+}
+
+function connectToLocalServer() {
+  Parse.initialize('testapp')
+  if (Platform.OS === 'android') {
+    Parse.serverURL = 'http://10.0.2.2:1337/parse'
+  } else {
+    Parse.serverURL = 'http://localhost:1337/parse'
```

Comment:
Let's place this string in a highly visible area.  Maybe a settings.js file or use a package like this https://www.npmjs.com/package/react-native-config